<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1527</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>821</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="loadGroupBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Загрузка</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QGroupBox" name="typeGroupBOx">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>89</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Представление</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QRadioButton" name="listRadioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Список</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="matrixRadioButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Матрица</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="fileSelectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>551</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать файл...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>321</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loadPushButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Загрузить</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="fileGroupBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>90</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Вид Файла</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QRadioButton" name="sqliteRadioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Sqlite</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="txtRadioButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Txt</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="postgresqlRadioButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>PostgreSQL</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cameraGroupBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>210</y>
      <width>571</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Камеры</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QDoubleSpinBox" name="cameraXSpin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-500.000000000000000</double>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="cameraZSpin">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>50</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-500.000000000000000</double>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="cameraYSpin">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>161</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-500.000000000000000</double>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="cameraAddPushbutton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QListWidget" name="cameraListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>281</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="cameraSetPushbutton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Установить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cameraDeletePushbutton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>120</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <!-- <widget class="QPushButton" name="cameraMoveButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>150</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Переместить</string>
     </property>
    </widget> -->
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>30</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="objectGroupBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>410</y>
      <width>571</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Объекты</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <widget class="QListWidget" name="objectListWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>281</width>
       <height>181</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="obkectMoveXSpin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-500.000000000000000</double>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectScaleXSpin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectRotateXSpin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectMoveYSpin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-490.000000000000000</double>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectRotateYSpin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectScaleYSpin">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>370</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectMoveZSpin">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-490.000000000000000</double>
     </property>
     <property name="maximum">
      <double>500.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectRotateZSpin">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>310</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="objectScaleZSpin">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>370</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="objectDeletePushbutton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>80</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить выбранные</string>
     </property>
    </widget>
    <widget class="QPushButton" name="objectCompositePushbutton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Объединить в композит</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>dx</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>ax</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>kx</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>230</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>dy</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>350</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>ky</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>ay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>290</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>az</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>dz</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>350</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>kz</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="objectMovePushbutton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Переместить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="objectRotatePushbutton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>310</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Повернуть</string>
     </property>
    </widget>
    <widget class="QPushButton" name="objectScalePushbutton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>370</y>
       <width>151</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Отмасштабировать</string>
     </property>
    </widget>
    <widget class="QPushButton" name="undoLastTransformPushbutton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Отменить преобразование</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toggleFaceCullingPushbutton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Переключить сокрытие граней</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
